### ISSUE TITLE
Weird results for test error and training error

### ISSUE THREAD

**Issue by vaastav** [Hanjun-Dai/pytorch_structure2vec#22]
Hi,
Sorry for bugging so much. I was trying to run the graph classification code with a custom dataset. The custom dataset is simple such that it contains 1000 copies of the same graph as I wanted to get a baseline for the graphs that I have. I expected the model to overfit and the training and test error to go to 0 over epochs but I am seeing something completely different. The test error never hits 0 but it starts increasing. Attached is the graph : (x-axis is the epoch #, blue line is training error, red line is test error).
I am wondering if I screwed something up in configuration and was wondering if you had any insight as to why I am seeing these results.

![losses](https://user-images.githubusercontent.com/6758381/57780819-4ebec380-7729-11e9-9ed7-5e1a896e25c4.png)
